packages/kubernetes-1.17/kubelet-kubeconfig

```diff
@@ -26,6 +26,8 @@ users:
       - token
       - "-i"
       - "{{settings.kubernetes.cluster-name}}"
+      - --region
+      - "{{settings.aws.region}}"
```

Comment:
It's possible to set the auth mode to "aws" for platforms other than AWS, and although it wouldn't work today, it might in the future.

So it'd be good to guard this with a check that the rendered setting is defined:
```suggestion
{{#if settings.aws.region}}
      - --region
      - "{{settings.aws.region}}"
{{/if}}
```

Reply:
Sounds good, thanks for the feedback!